code/extensions/che-commands/src/taskProvider.ts

```diff
@@ -37,7 +37,7 @@ export class DevfileTaskProvider implements vscode.TaskProvider {
 
 		const cheTasks: vscode.Task[] = devfileCommands!
 			.filter(command => command.exec?.commandLine)
-			.filter(command => !command.attributes || (command.attributes as any)['controller.devfile.io/imported-by'] !== 'editor')
+			.filter(command => !command.attributes || (command.attributes as any)['controller.devfile.io/imported-by'] === undefined)
```

Comment:
Another case that just occurred to me that we may want to cover -- there's a special case when devfiles use `parent`s. In this case, the command would have the attribute `controller.devfile.io/imported-by: "parent"` (hard-coded, by DWO). Should we include those commands? 

E.g. see the example in this issue: https://github.com/eclipse/che/issues/22103 -- we likely want to include the commands from the [java-maven](https://registry.devfile.io/devfiles/java-maven) parent.

Reply:
Good catch. Thanks @amisevsk !
I've prepared a fix in the draft PR https://github.com/che-incubator/che-code/pull/197/files. Will test it once the dogfoding instance is fixed (https://github.com/eclipse/che/issues/22108) and merge.